gluoncv/model_zoo/efficientnet.py

```diff
@@ -0,0 +1,784 @@
+
+    def forward(self, F, x):
+        ih, iw = x.shape[-2:]
+        kh, kw = self.kernel_size
+        sh, sw = self.stride
+        oh, ow = math.ceil(ih / sh), math.ceil(iw / sw)
+        pad_h = max((oh - 1) * self.stride[0] +
+                    (kh - 1) * self.dilation[0] + 1 - ih, 0)
+        pad_w = max((ow - 1) * self.stride[1] +
+                    (kw - 1) * self.dilation[1] + 1 - iw, 0)
+        if pad_h > 0 or pad_w > 0:
+            x = F.pad(x, mode='constant', pad_width=(0, 0, 0, 0, pad_w//2, pad_w -pad_w//2,
+                                                     pad_h//2, pad_h - pad_h//2))
+            return x
+        return x
+
+
+def _add_conv(out, channels=1, kernel=1, stride=1, pad=0,
+              num_group=1, active=True, batchnorm=True):
+    out.add(SamePadding(kernel, stride, dilation=(1, 1)))
+    out.add(nn.Conv2D(channels, kernel, stride, pad, groups=num_group, use_bias=False))
+    if batchnorm:
+        out.add(nn.BatchNorm(scale=True, momentum=0.99, epsilon=1e-3))
+    if active:
+        out.add(nn.Swish())
+
+
+class MBConv(nn.Block):
+    def __init__(self, in_channels, channels, t, kernel, stride, se_ratio=0,
+                 drop_connect_rate=0, **kwargs):
+
+        r"""
+            Parameters
+            ----------
+            int_channels: int, input channels.
+            channels: int, output channels.
+            t: int, the expand ratio used for increasing channels.
+            kernel: int, filter size.
+            stride: int, stride of the convolution.
+            se_ratio:int, ratio of the squeeze layer and excitation layer.
+            drop_connect_rate: int, drop rate of drop out.
+        """
+        super(MBConv, self).__init__(**kwargs)
+        self.use_shortcut = stride == 1 and in_channels == channels
+        self.se_ratio = se_ratio
+        self.drop_connect_rate = drop_connect_rate
+        with self.name_scope():
+            self.out = nn.Sequential(prefix="out_")
+            with self.out.name_scope():
+                if t != 1:
+                    _add_conv(
+                        self.out,
+                        in_channels * t,
+                        active=True,
+                        batchnorm=True)
+                _add_conv(
+                    self.out,
+                    in_channels * t,
+                    kernel=kernel,
+                    stride=stride,
+                    num_group=in_channels * t,
+                    active=True,
+                    batchnorm=True)
+            if se_ratio:
+                num_squeezed_channels = max(1, int(in_channels * se_ratio))
+                self._se_reduce = nn.Sequential(prefix="se_reduce_")
+                self._se_expand = nn.Sequential(prefix="se_expand_")
+                with self._se_reduce.name_scope():
+                    _add_conv(
+                        self._se_reduce,
+                        num_squeezed_channels,
+                        active=False,
```

Comment:
I believe there should be activation function in SE module.

Reply:
I follow the pytorch efficientnet while it doesn't have activation function in SE block.

Reply:
The `relu_fn` is `swish`
https://github.com/lukemelas/EfficientNet-PyTorch/blob/master/efficientnet_pytorch/model.py#L81